tests/integration/builders/builders_test.go

```
@@ -0,0 +1,258 @@
+			unique := uniqueStrings(tc.versions)
+			g.Expect(unique).To(HaveLen(tc.expectedUnique))
+		})
+	}
+}
+
+func extractGolangVersionsFromDockerfile(path string) ([]string, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open %s: %w", path, err)
+	}
+	defer file.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `file.Close` is not checked

(errcheck)